src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -472,6 +470,32 @@ public boolean expire()
                 .deRegisterRecurringRunnable(bandwidthProbing);
         }
 
+        MediaStream mediaStream = getStream();
+        if (mediaStream instanceof VideoMediaStream)
+        {
+            BandwidthEstimator.Statistics bweStats
+                = ((VideoMediaStream) mediaStream)
+                .getOrCreateBandwidthEstimator().getStatistics();
+            bweStats.update(System.currentTimeMillis());
+
+            Videobridge.Statistics videobridgeStats
+                = getContent().getConference().getVideobridge()
+                .getStatistics();
+
+            long lossLimitedSeconds = bweStats.getLossLimitedMs() / 1000;
+            long lossDegradedSeconds = bweStats.getLossDegradedMs() / 1000;
+            long participantSeconds = bweStats.getLossFreeMs() / 1000
+                + lossDegradedSeconds + lossLimitedSeconds;
+
+            videobridgeStats.totalParticipantSeconds
```

Comment:
This will fail to count audio only participants (e.g. jigasi)

Reply:
Ah, right :( in this PR I care about the loss-controlled seconds so I renamed the variable (we'll have to wait for the true participant-seconds).

Reply:
Aww, I was excited about it :)